scripts/install

```diff
@@ -46,9 +46,9 @@ ynh_app_setting_set $app adminusername $admin
 ynh_app_setting_set $app is_public $is_public
 
 # Create tmp directory and install app inside
-TMPDIR=$(ynh_mkdir_tmp)
+TMPDIR=`mktemp -d`
```

Comment:
I'm a little picky here but could you use $(command) instead of backquotes theses are deprecated. Also consistency with the rest of the script.